### PR TITLE
feat: Added Swagger documentation

### LINKED ISSUE #3: Add swagger docs support
**Is your feature request related to a problem? Please describe.**
It is great to have all API docs in one place. Especially if it is interactive and provides examples on response, request body etc.,

**Describe the solution you'd like**
Integrate swagger docs into the django server with help of django-yasg package.

**Additional context**
One may refer the <a href="https://github.com/ECellNitrr/teesco-backend/pull/53">PR made in teesco-backend</a> for adding swagger docs, if required.

### PR DESCRIPTION
**Pull request by sahil9001**
### Description
Added Swagger docs for the APIs for better navigation of routes.

**Type of Change:** (Delete irrelevant options)
- Code
- Documentation

**Code/Quality Assurance Only** (Delete irrelevant options)

- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Tested manually 

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [x] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: iportal/views.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ class ApplicantOrganisationViewset(ModelViewSet):
     queryset = ApplicantOrganisation.objects.all()
     serializer_class = ApplicantOrganisationSerializer
     filter_backends = (filters.DjangoFilterBackend,)
-    filterset_fields = ('applicant','organisation_name','role')    
+    filterset_fields = ('applicant','role')
```
Comment (anshsrtv) [inline iportal/views.py:95]: No changes in the I-Portal are required. It should be left as it is. Please restrain from making any changes to it.

**Review by anshsrtv — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -27,36 +29,47 @@ django-ses==0.8.14
...
+Jinja2==2.11.2
 jmespath==0.10.0
 kombu==4.6.8
 Markdown==3.1.1
+MarkupSafe==1.1.1
 msgpack==0.6.1
 mysqlclient==1.4.2.post1
+packaging==20.8
 Pillow==6.0.0
+pkg-resources==0.0.0
```
Comment (priyanshisharma) [inline requirements.txt:51]:
```suggestion
```
Remove this line. There is no matching distribution.
Turns out its bug crept in by Ubuntu. <a href="https://github.com/pypa/pip/issues/4022">Read here </a>to learn more about it.

File: iportal/views.py (hunk 1 above)
Comment (sahil9001) [reply iportal/views.py:95]: It's causing an issue, Naveen sir said that too

**Review by priyanshisharma — CHANGES_REQUESTED** [review]
Good work 🙌

File: requirements.txt (hunk 2 above)
Comment (sahil9001) [reply requirements.txt]: Yeah, I forgot that, Thanks ma'am!

**Review by priyanshisharma — APPROVED** [review]

**Review by anshsrtv — APPROVED** [review]

**Pull request merged**